### PR TITLE
Ensure that the RSpec test suites are passing for the absolute-ids Branch

### LINKED ISSUE #82: Restore RuboCop style compliance for the `absolute-ids` branch
This blocks the branch protection rule status check for RuboCop.

### LINKED ISSUE #83: Restore RSpec test suites for the `absolute-ids` branch
Like #82, this should be in place for proper branch protection for `absolute-ids`.

### PR DESCRIPTION
**Pull request by jrgriffiniii**
Resolves #82 and #83

### CODE REVIEW

**Comment by jrgriffiniii** [comment]
https://github.com/pulibrary/lib_jobs/pull/90/commits/1878509fbee2b8692f9c2618f01972c136a71d0f and https://github.com/pulibrary/lib_jobs/pull/90/commits/30a6be4fe058d7db99090d268f7a6136b06a683f resolved this. I still could not reproduce this locally, and should note that I am using the same Ruby version, as well as SQLite in both environments.

Perhaps the version of SQLite is the problem with this? For my MacBook, I am using release `SQLite version 3.32.3`, while on CircleCI, the release `SQLite version 3.27.2` is being used.

File: app/controllers/absolute_ids_controller.rb [hunk 1]
```diff
@@ -135,7 +131,7 @@ def synchronize
     @absolute_ids = @session.absolute_ids
 
     @absolute_ids.each do |absolute_id|
-      #ArchivesSpaceSyncJob.perform_later(user_id: current_user.id, model_id: absolute_id.id)
+      # ArchivesSpaceSyncJob.perform_later(user_id: current_user.id, model_id: absolute_id.id)
```
Comment (tpendragon) [inline app/controllers/absolute_ids_controller.rb:134]: Can you delete this commented code?

File: app/jobs/archives_space_sync_job.rb [hunk 2]
```diff
@@ -101,19 +101,15 @@ def client_mapper
   end
 
   def update_top_container(uri:, barcode:, indicator:, location:)
-    #sync_client = client_mapper.sync_client
+    # sync_client = client_mapper.sync_client
```
Comment (tpendragon) [inline app/jobs/archives_space_sync_job.rb:104]: Delete commented code

File: app/models/absolute_id/resource_child_record.rb [hunk 3]
```diff
@@ -6,8 +6,9 @@ def self.build_from_resource(resource)
     resource_attributes = resource.attributes.deep_dup
 
     resource_attributes.delete(:children)
-    resolved_children = resource.resolve_children
-    #children = child_resources.map(&:find_or_create_model)
+    # This needs to implement polymorphic assocations for child Resources
+    # resolved_children = resource.resolve_children
```
Comment (tpendragon) [inline app/models/absolute_id/resource_child_record.rb:10]: Delete commented code.

**Review by tpendragon — CHANGES_REQUESTED** [review]
Glad to see the suite passing! Just some comments about deleting some left over commented code.

**Review by tpendragon — APPROVED** [review]

**Pull request merged**